.travis.yml

```diff
@@ -32,7 +32,9 @@ install:
   - npm install && npm run lint
 
 script:
-  - npm run build_spec && npm run test_mocha && node ./node_modules/markdown-doctest/bin/cmd.js
+  - npm install typescript@2.0.7 && npm run build_spec && npm run test_mocha
+  - npm install typescript@latest && npm run build_spec && npm run test_mocha
```

Comment:
this is kind of hacky, monkey patching to test on TS@2.0.x and latest both. Maybe better to setup different CI configuration to test each compilers instead in further. 

Note : obviously this will slow down CI build.